Sofee.Core.Api.Tests.Unit/Services/Foundations/LanguageServiceTests.cs

```diff
@@ -0,0 +1,62 @@
+﻿// -------------------------------------------------------
+// Copyright (c) Coalition of the Good-Hearted Engineers
+// FREE TO USE FOR THE WORLD
+// -------------------------------------------------------
+
+using System;
+using System.Linq.Expressions;
+using Moq;
+using Sofee.Core.Api.Brokers.Loggings;
+using Sofee.Core.Api.Brokers.Storages;
+using Sofee.Core.Api.Models.Languages;
+using Sofee.Core.Api.Services.Foundations.Languages;
+using Tynamix.ObjectFiller;
+using Xeptions;
+
+namespace Sofee.Core.Api.Tests.Unit.Services.Foundations
+{
+    public partial class LanguageServiceTests
+    {
+        private readonly Mock<IStorageBroker> storageBrokerMock;
+        private readonly Mock<ILoggingBroker> loggingBrokerMock;
+        private readonly ILanguageService languageService;
+
+        public LanguageServiceTests()
+        {
+            this.storageBrokerMock = new Mock<IStorageBroker>();
+            this.loggingBrokerMock = new Mock<ILoggingBroker>();
+
+            this.languageService = new LanguageService(
+                storageBroker: this.storageBrokerMock.Object,
+                loggingBroker: this.loggingBrokerMock.Object);
+        }
+
+        private static Language CreateRandomLanguage() =>
+                CreateLanguageFiller().Create();
+
+        private static Filler<Language> CreateLanguageFiller()
```

Comment:
This method to be the last in this class.

Reply:
done